src/exec.rs

```diff
@@ -62,6 +62,44 @@ pub fn exec_batch_proofs(
     use_ecc_select_chip: bool,
     open_schema: OpenSchema,
     accumulator: Accumulator,
+) {
+    exec_batch_proofs_with_circuit_names(
+        params_cache,
+        pkey_cache,
+        proof_name,
+        &format!("{}.start", proof_name),
+        &format!("{}.rec", proof_name),
+        &format!("{}.final", proof_name),
+        output_dir,
+        params_dir,
+        config_files,
+        commits,
+        hash,
+        k,
+        cont,
+        use_ecc_select_chip,
+        open_schema,
+        accumulator,
+    );
+}
+
+pub fn exec_batch_proofs_with_circuit_names(
```

Comment:
Xin suggest we do not just wrap the function like this.
This is because only "continuation" feature need the additional names args to specify circuit data name. However, if not continuation feautre (we call it flat), in fact the args are not need.

So to fix it properly, we should seperate the continuation exec batch proofs and flat batch proofs.

So in order to work it properly for logic, we'd better do it like this.
We need seperate the functions for cont and flat batch.
exec_batch_proofs_continuation(...,proof_name, start_circuit_prefix: &String, rec_circuit_prefix: &String, final_circuit_prefix: &String,...)

exec_batch_proofs_flat(...,proof_name,...)

and exec_batch_proofs_continuation do cont is number scenario and exec_batch_proofs_flat do cont is None scenario. (The input params no need cont any more)

For backward compatibility, we keep the exec_batch_proofs()
and its content is like
`
if cont.is_some() { exec_batch_proofs_continuation }
else {exec_batch_proofs_flat }
`
We can also have an additional common function for "generate aux data for proof" which can be called in both function to do the last step in the original exec_batch_proofs() so no need 2 copies of the codes